Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -397,7 +401,7 @@ class BrowserViewController: UIViewController {
 
     private func updateInContentHomePanel(url: NSURL?) {
         if !urlBar.isEditing {
-            if url == AppConstants.AboutHomeURL {
+            if AboutUtils.isAboutHomeURL(url!){
```

Comment:
Since the argument to `updateInContentHomePanel()` is of type `NSURL?`, that means it can be `nil`, so we don't want to unwrap it here or else we'll crash. Instead, make sure `url` is non-nil before we try to use it.